.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
Weren't we removing this? Please remove. This is going to cost me GBs.

Reply:
I thought I removed it